src/Protocol/SymbolInformation.php

```diff
@@ -58,13 +58,25 @@ public static function fromNode(Node $node, string $fqn = null)
             Node\Stmt\PropertyProperty::class => SymbolKind::PROPERTY,
             Node\Const_::class                => SymbolKind::CONSTANT
         ];
-        $class = get_class($node);
-        if (!isset($nodeSymbolKindMap[$class])) {
-            throw new Exception("Not a declaration node: $class");
-        }
         $symbol = new self;
-        $symbol->kind = $nodeSymbolKindMap[$class];
-        $symbol->name = (string)$node->name;
+        if (
+            $node instanceof Node\Expr\FuncCall
+            && $node->name instanceof Node\Name
+            && (string)$node->name === 'define'
```

Comment:
I'll check this, but it should be excluded with `$node->name instanceof Node\Name`